src/features/FeatureExtractor.js

```diff
@@ -448,8 +448,13 @@ class FeatureExtractor {
 
         for await (const dumpLine of dumpReadLineI) {
             const requestSize = sizeof(dumpLine);
-            const dumpLineObj = JSON.parse(dumpLine);
+            let dumpLineObj;
 
+            try {
+                dumpLineObj = JSON.parse(dumpLine);
```

Comment:
Not sure if we want to continue here, if the dump is corrupted in one way or another that might mean the resulting aggregation is tainted as well, e.g. a stats entry has a bad format and we ignore it here, the next one is delta compressed based on the previous one (corrupted entry) resulting in unexpected behavior. I propose we just fail here instead of trying to recover.